reblog/_config.yml

```diff
@@ -0,0 +1,19 @@
+# Book settings
+title: Bayesnets
+author: Martinis and Research
+logo: martini.png
+
+html:
+  favicon: favicon.ico
+  use_repository_button     : true  # Whether to add a link to your repository button
+
+latex:
+  latex_documents:
+    targetname: book.tex
+
+repository:
+  url: https://github.com/martinisandresearch/bayesnets
+  path_to_book: reblog
+
+execute:
+  timeout: 1200
```

Comment:
This looks like more magic in yml!   Keen to understand better.

Reply:
There is so much copy paste haha. Yaml is all just config. Jupyter book created this and I just modified parts here and there looking at various versions